app/dose-event-card/dose-event-card.component.html

```diff
@@ -25,12 +25,12 @@
         <ion-col>
 
             <button primary clear small>
-                <ion-icon name="md-thumbs-up"></ion-icon>
+                <ion-icon name="thumbs-up"></ion-icon>
                 <div>12 Likes</div>
             </button>
 
             <button primary clear small>
-                <ion-icon name="md-text"></ion-icon>
+                <ion-icon name="text"></ion-icon>
```

Comment:
Now that the whole app is operating in Material Design mode, I can remove explicitly using md icons.